client/src/pages/ModalBookingScreen.js

```diff
@@ -135,12 +134,8 @@ function ModalBookingScreen({
 										/>
 									</Form.Field>
 									<Divider inverted />
-									<Form.Field inline>
-										<Popup
-											content={formatBookingDate(bookingDate)}
-											trigger={<Icon name="calendar alternate outline" />}
-										/>
-										<Input
+									<Form.Field>
+										<Form.Input
```

Comment:
Good it looks fixed now! 
thanks Oneil.

Reply:
It's all good Benjamin 👍🏼